feg/cloud/go/services/feg_relay/servicers/create_bearer_pgw.go

```diff
@@ -36,11 +36,11 @@ func (srv *FegToGwRelayServer) CreateBearer(
 	if err := validateFegContext(ctx); err != nil {
 		return nil, err
 	}
-	if req == nil || req.BearerContext == nil || req.BearerContext.UserPlaneFteid == nil {
+	if req == nil {
 		glog.Errorf("unable to send CreateBearerPGWUnverified, Teid not found  %v", req)
```

Comment:
you know, req == nil here, what's a point in Errorf & %v?
Just do glog.Error("unable to send CreateBearerPGWUnverified, nil request")

Reply:
Before it made sense when the conditions were more. Now it does not.